### PR TITLE
Update documentation and add postcss-pseudo-classes as dependency

### PR DESCRIPTION
**Pull request by jzipfler**
Besides of the documentation update, I moved the *postcss-pseudo-classes* package to the *dependencies* section.

If you want to have this change in a separate pull request I can move it back again so that this PR consists only of documentation changes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,12 @@
   },
   "author": "philippone",
   "license": "MIT",
+  "dependencies": {
+    "postcss-pseudo-classes": "0.3.0"
```
Comment (philippone) [inline package.json:58]:
```suggestion
    "postcss-pseudo-classes": "^0.3.0"
```

File: package.json [hunk 2]
```diff
@@ -54,6 +54,12 @@
...
+  },
+  "peerDependencies": {
+    "postcss-loader": "4"
```
Comment (philippone) [inline package.json]:
```suggestion
    "postcss-loader": "^4"
```

**Review by philippone — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (jzipfler) [reply package.json]: `4` und `^4` bedeuten das gleiche. Kann aber verstehen, dass `^4` besser da expliziter ist. 👍

**Comment by felquis** [comment]
Is this related to this PR?
"@ergosign/storybook-addon-pseudo-states-react": "0.0.1-alpha.42",
```
ERROR in ./node_modules/@ergosign/storybook-addon-pseudo-states-react/dist/share/preset-utils.js
Module not found: Error: Can't resolve 'postcss-pseudo-classes' in '/node_modules/@ergosign/storybook-addon-pseudo-states-react/dist/share'
```

**Comment by jzipfler** [comment]
> Is this related to this PR?
> "@ergosign/storybook-addon-pseudo-states-react": "0.0.1-alpha.42",
> 
> ```
> ERROR in ./node_modules/@ergosign/storybook-addon-pseudo-states-react/dist/share/preset-utils.js
> Module not found: Error: Can't resolve 'postcss-pseudo-classes' in '/node_modules/@ergosign/storybook-addon-pseudo-states-react/dist/share'
> ```

As you provided no context about the error, I can only assume that you may get this error when adding this addon to your storybook configuration and try to start it immediately without installing the *postcss-pseudo-classes* package on your own.

If thats the case, then this PR could solve this problem.

An other thing to remember is, that now the *postcss-loader* is now a peer dependency. So you may have to take care for that, if your setup does not actually includes this.

**Pull request merged**